install-manuals/all-services/README.md

```diff
@@ -1,21 +1,20 @@
 # How to run all services
 
-All services are in a `docker compose` file.
+All services are available as docker containers. The preferred way to host is to use a `docker compose` file.
 
-## Prerequisite
+We provide a template in our [run-config](https://github.com/Gamify-IT/run-config) repository. There is also a [manual](https://github.com/Gamify-IT/run-config/hosting-template#setup) how to setup the hosting environment.
 
-### Install docker
 
-[Install on windows](https://docs.docker.com/desktop/windows/install/)
+## Prerequisite
 
-[Install on linux](https://docs.docker.com/desktop/linux/install/)
+### Install docker
 
-[Install on macos](https://docs.docker.com/desktop/mac/install/)
+[Istall Docker](https://docs.docker.com/engine/install/)
 
-### Clone project
+#### Docker Desktop
 
-Clone `https://github.com/Gamify-IT/prototype`
+[Install Docker Desktop on windows](https://docs.docker.com/desktop/windows/install/)
 
-## Run project
+[Install Docker Desktop on linux](https://docs.docker.com/desktop/linux/install/)
 
-Run in the directory of the cloned project `docker-compose up --build`
+[Install Docker Desktop on macos](https://docs.docker.com/desktop/mac/install/)
```

Comment:
1. `[I`**n**`stall Docker]`
2. `#### For developers: Docker Desktop`
3. Erm… Why is the `prerequisite` header the last header?  That goes against any design I've ever heard of. **Everyone** expects something after it as well, why else would it be a **pre**requisite?

Reply:
2. Why is Docker Desktop for developers? In my opinion the decision of using a GUI has nothing todo with that. 

3. I think docker is so fundamental that it don't has to be mentioned at the beginning. What about if we add a notice at the beginning and link the `Prerequisite` section there

Reply:
2. As I've recently heard: `You don't use Docker Desktop in Production. When you exit it, your deployments are killed as well…`.
3. I don't quite understand what you mean.

Reply:
2. Ok I never heard that. What I know is that Docker ~~Desktop~~ is hard to install on windows (Needs a VM or some advanced setup in WSL). What about we add a notice "Not recommended for production use"?

Reply:
Regarding 2: I think it was @robojones, who told me that.